backend/core/src/util/hash.rs

```diff
@@ -11,6 +12,13 @@ const HASH_BYTES: usize = 32;
 #[derive(Hash, PartialEq, Eq, Clone, Copy)]
 pub struct Hash([u8; HASH_BYTES]);
 
+impl Default for Hash {
+    #[inline(always)]
+    fn default() -> Hash {
```

Comment:
maybe make this const instead of `default impl` such as `const fn new/zero`? Such as https://docs.rs/ethereum-types/0.12.0/ethereum_types/struct.H256.html#method.zero